### PR TITLE
Rephrase default value advice

### PR DESCRIPTION
**Pull request by acparas**
- The original document asserted that "Short-circuiting is cleaner than conditionals"
- ...only to override this assertion later with "Use default arguments instead of short circuiting."
- This is a poor design-choice for an advisory document.

    - If you are going to say x > y,
    - then later say z > x,
    - you should just say z > x > y upfront.
    - Otherwise, many people might not see the z > x later on,
    - because many people will probably use this as a referential document. They probably won't read the entire work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,25 +188,36 @@ function paintCar(car) {
...
     breweryName = name;
   } else {
     breweryName = 'Hipster Brew Co.';
   }
+  ...
 }
 ```
 
-**Good**:
+**Better:**
```
Comment (ryanmcdermott) [inline README.md]: Thanks for more contributions!

I agree with you in principle. It does seem to me that people are going to read this non-linearly. I do give advice in a linear way of increasing cleanliness -- see the Concurrency section. If we set the precedent here, that would need to be fixed.

As for your solution, this is looking almost perfect. I don't want to change the format of this project though, so let's avoid having a Better clause. Can we just have Good/Bad?

**Comment by acparas** [comment]
I addressed your request. How does that look now?

Oh, and a big thanks for adapting *Clean Code* to JavaScript + being considerate to all the pull requests!

**Comment by ryanmcdermott** [comment]
That's almost there! I'm sorry I wasn't very clear with my thoughts. Can you have just one Bad example and one Good example? Thanks!

**Comment by acparas** [comment]
I changed it to one Bad example and one Good example. How's that look now?

**Comment by ryanmcdermott** [comment]
That's great, thank you for your hard work!

**Pull request merged**